app/src/main/java/com/quran/labs/androidquran/util/ShareUtil.kt

```diff
@@ -124,4 +128,15 @@ class ShareUtil @Inject internal constructor(private val quranDisplayData: Quran
       append("]")
     }
   }
+
+
+  fun shareAudioFileIntent(activity: Activity, file: File) {
+    val authorities = BuildConfig.APPLICATION_ID + ".fileprovider"
+    val uri = FileProvider.getUriForFile(activity, authorities, file)
+    val shareIntent = Intent(Intent.ACTION_SEND)
+    shareIntent.putExtra(Intent.EXTRA_STREAM, uri)
+    shareIntent.type = "audio/mp3"
+    shareIntent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION)
+    activity.startActivity(Intent.createChooser(shareIntent, "Share"))
```

Comment:
Instead of "Share", You could use one of the following (and don't forget to make it localizable)
- Share Audio File
- Share Recitation of Husary
- Share Recitation of Husary for 1:1 - 1:5

Reply:
> * Share Audio File

Done